### DISCUSSION TITLE
How to fix error when gatsby netlify build?

### DISCUSSION THREAD

**Discussion by topstarwebking** [community/community#20849] [Other Feature Feedback, Questions, & Ideas] [answered]
I had an error when I build project(gatsby) on Netlify.
Font issue: they cannot build font family.

Please help me.

**Comment by devlancer412** [comment, marked as answer]
I solved similar problem in the last.
I think you can change casesensitive to lowercasesensitive.
I hope it would be help you.

**Comment by johnsonmake** [comment]
You should have this choice if you navigate to Netlify dashboard > Deploys and select the "Trigger Deploy" dropdown. Try eliminating the yarn if that doesn't work. To force Netlify to pull brand-new copies of everything, lock and install dependencies once more. A similar issue cropped up for me when I was working on [Trendsza ](https://trendsza.com/)but trigger deploys fixed it.
